### DISCUSSION TITLE
(not) Numbering Content Nodes

### DISCUSSION THREAD

**Discussion by gielians** [VentuzTechnology/Ventuz.Extension.prerelease#30] [Ideas]
Why do the content nodes get numbers when you add them.
It is (in my opinion)

- confusing because if you copy a node the number stayes the same
- unnecessary: i don't even see the need for numbers at content nodes (its nice for containers but you cant build them anyway...)
- incoherent because none of the ventuz container nodes have numbers

In my opinion numbering content nodes should not be a thing for now

**Comment by VentuzRalfStanke** [comment]
The numbering is the default behavior for all nodes. You can change it by specifying a different "default name" with the VxToolboxAttribute. A "*" is the placeholder for the numbering. Just define a name without an "*"
